ui/index.d.ts

```diff
@@ -29,8 +30,9 @@ export declare class AzureTreeDataProvider implements TreeDataProvider<IAzureNod
      * @param ui Used to get input from the user
      * @param telemetryReporter Optionally used to track telemetry for the tree
      * @param rootTreeItems Any nodes other than the subscriptions that should be shown at the root of the explorer
+     * @param testAccount A test Azure Account validated with a service principal that is leveraged for CI testing.
```

Comment:
I don't understand "validated with a service principal"

How about "A test Azure Account that leverages a service principal instead of interactive login"